sagui.config.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * Sagui configuration object
+ */
+module.exports = {
+  /**
+   * Different application entry-points
+   * Each page is a combination of a JavaScript file and a HTML file
+   *
+   * Example: 'index' -> 'index.html' and 'index.js'
+   */
+  pages: ['index', 'source', 'target'],
```

Comment:
On tha same note, I want to add support to pages in nested folders, which would also make sense here:

```js
pages: ['examples/index', 'examples/source', 'examples/target']
```

Unfortunately that is not done yet. I'll try get it done this Sunday.